api/managementpb/backup/backups.proto

```diff
@@ -0,0 +1,66 @@
+syntax = "proto3";
+
+package backup.v1beta1;
+
+option go_package = "api/managementpb/backup;backupv1beta1";
+
+import "google/api/annotations.proto";
+import "google/protobuf/timestamp.proto";
+
+// Status shows current status of Backup.
+enum Status {
+  STATUS_INVALID = 0;
+  PENDING = 1;
+  IN_PROGRESS = 2;
+  PAUSED = 3;
+  SUCCESS = 4;
+  ERROR = 5;
+}
+
+// DataModel is a model used for performing a backup.
+enum DataModel {
+  DATA_MODEL_INVALID = 0;
+  PHYSICAL = 1;
+  LOGICAL = 2;
+}
+
+// Backup represents single Backup Location.
+message Backup {
```

Comment:
@oter What about backup type (incremental vs full)? Or do you mean we add it later?

Reply:
we will add it later